src/api.ts

```diff
@@ -248,3 +265,25 @@ export function cast(snapshotOrInstance: never): never;
 export function cast(snapshotOrInstance: any): any {
   return snapshotOrInstance;
 }
+
+/**
+ * Defines a new asynchronous action that uses `yield` instead of `await` for waiting for the results of promises
```

Comment:
I think would be helpful to also mention why you'd want to use flow with generators instead of async/await.

Or maybe just reference https://mobx.js.org/actions.html#using-flow-instead-of-async--await-?